src/Bicep.Core.RegistryClient/Bicep.Core.RegistryClient.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <Nullable>annotations</Nullable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
+    <!-- The below settings were originally included by AutoRest to pull in shared sources into the project via a content/build NuGet package -->
+    <!--<IncludeGeneratorSharedCode>true</IncludeGeneratorSharedCode>-->
+    <!--<RestoreAdditionalProjectSources>https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json</RestoreAdditionalProjectSources>-->
```

Comment:
>RestoreAdditionalProjectSources [](http://example.com/codeflow?start=9&length=31)

This seemed a bit sketchy to me, so I grabbed the content from the public nuget and checked it in.